R/examples.R

```diff
@@ -1,16 +1,37 @@
 #' Example ID
 #' @description
-#' This function returns examples of working IDs that can be used with the API.
+#' This function returns examples of working IDs that can be used with the eesyapi functions, such
+#' as query_dataset().
 #'
-#' @param level Level of ID example to return: "publication" or "data set"
-#' @param environment Environment to return a working example for
-#' @param group Choose the publication group of examples to use. Can be "attendance".
+#' @param level Level of ID example to return. A range are available, although not every example
+#' group necessarily contains all possible examples. The following are generally available.
+#'   - "all": Return full list of example options for given group.
+#'   - "publication": Return example publication ID
+#'   - "dataset": Return example data set ID
+#'   - "time_period" / "time_periods": Return example time_period(s)
+#'   - "location_id" / "location_ids": Return example location ID(s)
+#'   - "location_code": Return example location code
+#'   - "filter": Return example filter column ID
+#'   - "filter_item" / "filter_items_short" / "filter_items_long": Return example filter ID or
+#'     example short / long filter query list.
+#'   - "indicator": Return example indicator ID
+#' @param environment Environment to return a working example for: "dev" or "test"
+#' @param group Choose the publication group of examples to use. Options are:
+#'   - "attendance"
```

Comment:
Should mention something about size here? Like attendance is a great, big example and the other tends to be smaller

Reply:
Excellent